src/app/new-contact/new-contact.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { ContactsService } from '../contacts.service';
+
+@Component({
+  selector: 'app-new-contact',
+  templateUrl: './new-contact.component.html',
+  styleUrls: ['./new-contact.component.css']
+})
+export class NewContactComponent implements OnInit {
+  contactForm = new FormGroup({
+    firstName: new FormControl(''),
+    lastName: new FormControl(''),
+    street: new FormControl(''),
+    city: new FormControl(''),
+    email: new FormControl(''),
+  })
+
+  constructor(private contactsService: ContactsService){
+
+  }
+
+  ngOnInit(): void {
+    this.contactForm.controls.firstName.addValidators(Validators.required)
```

Comment:
Remember we can use `FormBuilder` and add the validators inside the definition to avoid this many lines of code.